packages/postgres-database/src/simcore_postgres_database/models/services.py

```diff
@@ -28,29 +35,46 @@
             ondelete="RESTRICT",
         ),
         nullable=True,
+        doc="Identifier of the group that owns this service",
+    ),
+    sa.Column("name", sa.String, nullable=False, doc="Display label"),
+    sa.Column(
+        "description", sa.String, nullable=False, doc="Markdown-compatible description"
+    ),
+    sa.Column(
+        "thumbnail",
+        sa.String,
+        nullable=True,
+        doc="Link to image to us as service thumbnail",
     ),
-    sa.Column("name", sa.String, nullable=False),
-    sa.Column("description", sa.String, nullable=False),
-    sa.Column("thumbnail", sa.String, nullable=True),
     sa.Column(
         "classifiers",
         ARRAY(sa.String, dimensions=1),
         nullable=False,
         server_default="{}",
+        doc="List of tags to describe this service (see classifiers table)",
```

Comment:
```suggestion
        doc="List of classifiers to describe this service (see classifiers table)",
```
sorry tags are something else no?